Assets/Scripts/ScreenShotScript.cs

```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+
+public class ScreenShotScript : MonoBehaviour
+{
+
+	public AudioSource shutterAudio;
+	public RawImage screenShot;
+	private Texture2D screenShotTexture;
+	private string lastScreenShotPath;
+	
+	// Use this for initialization
+	void Start ()
+	{
+		screenShotTexture = new Texture2D (100, 100);
+		screenShot.texture = screenShotTexture;
+	}
+	
+	// Update is called once per frame
+	void Update ()
+	{
+		
+	}
+
+	void OnGUI ()
+	{
+		Event e = Event.current;
+		
+		if (e.type == EventType.KeyDown && e.keyCode == KeyCode.F) {
+			string current_time = System.DateTime.Now.ToString ().Replace ("/", "_").Replace (":", "_");
+			lastScreenShotPath = Application.temporaryCachePath + "/" + current_time + ".png";
```

Comment:
That "Application.temporaryCachePath" returns a path for temporary usage. [reference](http://qiita.com/w_yang/items/8458cd790607d14b1b36)